numba/cuda/cudadecl.py

```diff
@@ -346,6 +347,19 @@ class Cuda_fp16_unary(ConcreteTemplate):
     return Cuda_fp16_unary
 
 
+def _genfp16_unary_operator(l_key):
+    @register
```

Comment:
```suggestion
    @register_global
```

Reply:
Change made although the key needed to be added as an argument.